common/placeholders.js

```diff
@@ -66,14 +67,25 @@ function getPlaceholders(types) {
   return placeholders
 }
 
-function table(value) {
-  let tableFinal = `<table><tr><td><p>Item</p></td><td><p>Value</p></td></tr>`
+function renderReceipt(values) {
+  const ids = ['organization', 'address', 'url', 'supportNumber', 'ein']
+  let table = '<table>'
+
+  for (let i = 0; i < ids.length; i++) {
+    const placeholder = placeholders.find(p => p.id === ids[i])
+    table += `<tr><td>${placeholder.label}:</td><td> ${values[ids[i]]}</td></tr>`
+  }
+
+  table += `<tr><td><p>Here you have a list of the donations our organization recibed from you:</p></td></tr>`
+  const items = values.items
+  table += `<tr><td><p>Item</p></td><td><p>Value</p></td></tr>`
   let total = 0
-  for (let i = 0; i < value.length; i++) {
-    tableFinal += `<tr><td><p>"${value[i].name}"</p></td><td><p>"${value[i].value}"</p></td></tr>`
-    total += value[i].value
+  for (let i = 0; i < items.length; i++) {
+    table += `<tr><td><p>"${items[i].name}"</p></td><td><p>"${items[i].value}"</p></td></tr>`
```

Comment:
Why the " 's around item name, value and total? Doesn't it look weird?

Reply:
What are you referring exactly??
I do not understand!

Reply:
Sorry! I don't think you need the double quotes, it should just be $11 not "$11".